Java/avg/array.java

```diff
@@ -12,4 +12,15 @@ public static void main(String[] args) {
             System.out.println(arr[i]);
         }
     }
+
+    public static void Even(String[] args) {
+        Scanner sc = new Scanner(System.in);
+        int[] arr = new int[]{1, 2, 3, 4, 5};

```

Comment:
**Suggestion**: Consider passing the array as a parameter to the method instead of hardcoding it within, to enhance reusability and facilitate unit testing. <br> **Code Suggestion**: 
 ```
 -        int[] arr = new int[]{1, 2, 3, 4, 5};
 ```